### DISCUSSION TITLE
Localhost: Domain

### DISCUSSION THREAD

**Discussion by destroyer743** [dubinc/dub#705] [Q&A]
How can I connect/configure my domain on Localhost?

It says :"Domain Not Found No redirect configured"

Under A Record (recommended) 
To configure your apex domain `undefined`, set the following A record on your DNS provider to continue:

and CNAME Record
To configure your subdomain `undefined`, set the following CNAME record on your DNS provider to continue:

my domain is showing as an undefined.  it should show my domain that i have added.

**Comment by bfzli** [comment]
@steven-tey you should answer key questions (besides the ones sharing the link created and similar stuff) to this to people who want to contribute to the project. Is there any way to run this on the localhost domain to create redirects and test them all locally? Or how to do this because this is not mentioned in the docs on how to start development.
